packages/front-end/components/Features/RuleList.tsx

```diff
@@ -60,6 +65,8 @@ export default function RuleList({
       coordinateGetter: sortableKeyboardCoordinates,
     })
   );
+  const showInactiveToggle =
+    items.filter((r) => !r.enabled || r.scheduleRules?.length).length > 0;
```

Comment:
If the intention of this is to only show the toggle when there is at least one rule that is inactive then it could be made a bit better as sometimes all of the scheduled rules will be active.  We have code somewhere that determines if a scheduled rule is currently active or not as we gray out the row when it is not, so ideally we would use the same logic here.

Reply:
I was thinking that the numbers might help the user understand that there are hidden rules there - but thinking this might not be an important distinction. I've refactored this a bit more to address these concerns, and moving the filtering up to the list level.